components/eamxx/src/physics/nudging/eamxx_nudging_process_interface.cpp

```diff
@@ -125,18 +138,57 @@ void Nudging::initialize_impl (const RunType /* run_type */)
 {
   using namespace ShortFieldTagsNames;
 
-  // Initialize the time interpolator
-  auto grid_ext = m_grid->clone(m_grid->name(), false);
-  grid_ext->reset_num_vertical_lev(m_num_src_levs);
+  // Initialize the refining remapper stuff at the outset,
+  // because we need to know the grid information.
+  // For now, we are doing the horizontal interpolation last,
+  // so we use the m_grid (model physics) as the target grid
+  grid_int = m_grid->clone(m_grid->name(), false);
```

Comment:
I think you want the last argument to be `true`, since I think shallow copy is enough for you.